test/lower-non-standard-types.ll

```diff
@@ -0,0 +1,58 @@
+; RUN: llvm-as %s -o %t.bc
+; RUN: llvm-spirv -s %t.bc -o - | llvm-dis -o - | FileCheck %s
+
+; CHECK: %0 = addrspacecast <3 x i64> addrspace(1)* @Id to <3 x i64> addrspace(4)*
+; CHECK: %1 = load <3 x i64>, <3 x i64> addrspace(4)* %0, align 32
+; CHECK: %2 = extractelement <3 x i64> %1, i64 0
+; CHECK: %3 = trunc i64 %2 to i32
+; CHECK: %4 = load <3 x i64>, <3 x i64> addrspace(4)* %0, align 32
+; CHECK: %5 = extractelement <3 x i64> %4, i64 2
+; CHECK: %6 = trunc i64 %5 to i32
+; CHECK: %conv1 = sitofp i32 %3 to float
+; CHECK: %conv2 = sitofp i32 %6 to float
+
+; CHECK-NOT: %0 = bitcast <3 x i64> addrspace(1)* @Id to <6 x i32> addrspace(1)*
+; CHECK-NOT: %1 = addrspacecast <6 x i32> addrspace(1)* %0 to <6 x i32> addrspace(4)*
+; CHECK-NOT: %2 = load <6 x i32>, <6 x i32> addrspace(4)* %1, align 32
+; CHECK-NOT: %3 = load <6 x i32>, <6 x i32> addrspace(4)* %1, align 32
+; CHECK-NOT: %4 = extractelement <6 x i32> %2, i32 0
+; CHECK-NOT: %5 = extractelement <6 x i32> %3, i32 4
+; CHECK-NOT: %conv1 = sitofp i32 %4 to float
+; CHECK-NOT: %conv2 = sitofp i32 %5 to float
```

Comment:
I think that it is enough to check that there is no "<6 x i32>" string in a result.